app/api.anatroc/src/AppBundle/Api/Subway/SubwayTCL.php

```diff
@@ -62,7 +62,7 @@ public function getGuzzle(): Client
 
     public function getStations()
     {
-        $response = $this->getGuzzle()->get('https://download.data.grandlyon.com/ws/rdata/tcl_sytral.tclarret/all.json');
+        $response = $this->getGuzzle()->request("GET",'https://download.data.grandlyon.com/ws/rdata/tcl_sytral.tclarret/all.json');
```

Comment:
En théorie c'est la même chose, après on a pas déterminer pourquoi la 1ère méthode ne fonctionnait pas.
Perso j'utilise la seconde et magique dans ce cas it works XD